app/views/posts/show.html.erb

```diff
@@ -4,7 +4,10 @@
     <p class="post-likes-comments">Comments: <%= @post.comment_counter %>, Likes: <%= @post.like_counter %></p>
     <p><%= @post.text %></p>
```

Comment:
I will leave the comment here since you do not have any changes on the `posts/new.html.erb` file.

### Create new post authorization
It would be best if you fixed the `Create Post` button so that users are able to create posts successfully. In my case, I was unsuccessful in creating new posts for the current/logged-in user. Either something is wrong with your authorization or the `create` posts actions. Kindly look into it.

![image](https://user-images.githubusercontent.com/38088818/232014841-1902fedc-7718-42e5-b4cc-a0ae189b415c.png)

- Since I cannot create a new post, I'm unable to verify if the author of the post can delete the newly created post. When this issue is resolved, kindly make sure you check and make sure that the owner can delete their post.